ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/UpdateAcademyConversionProjectPageModel.cs

```diff
@@ -1,116 +1,120 @@
+				RecommendationForProject = AcademyConversionProject.RecommendationForProject,
+				Author = AcademyConversionProject.Author,
+				ClearedBy = AcademyConversionProject.ClearedBy,
+				AcademyOrderRequired = AcademyConversionProject.AcademyOrderRequired,
+				ProposedAcademyOpeningDate = AcademyConversionProject.ProposedAcademyOpeningDate,
+				SchoolAndTrustInformationSectionComplete = AcademyConversionProject.SchoolAndTrustInformationSectionComplete,
+				PublishedAdmissionNumber = AcademyConversionProject.PublishedAdmissionNumber,
+				ViabilityIssues = AcademyConversionProject.ViabilityIssues,
+				FinancialDeficit = AcademyConversionProject.FinancialDeficit,
+				IsThisADiocesanTrust = AcademyConversionProject.IsThisADiocesanTrust,
+				DistanceFromSchoolToTrustHeadquarters = AcademyConversionProject.DistanceFromSchoolToTrustHeadquarters,
+				DistanceFromSchoolToTrustHeadquartersAdditionalInformation = AcademyConversionProject.DistanceFromSchoolToTrustHeadquartersAdditionalInformation,
+				GeneralInformationSectionComplete = AcademyConversionProject.GeneralInformationSectionComplete,
+				SchoolPerformanceAdditionalInformation = AcademyConversionProject.SchoolPerformanceAdditionalInformation,
+				RationaleForProject = AcademyConversionProject.RationaleForProject,
+				RationaleForTrust = AcademyConversionProject.RationaleForTrust,
+				RationaleSectionComplete = AcademyConversionProject.RationaleSectionComplete,
+				RisksAndIssues = AcademyConversionProject.RisksAndIssues,
+				RisksAndIssuesSectionComplete = AcademyConversionProject.RisksAndIssuesSectionComplete,
+				RevenueCarryForwardAtEndMarchCurrentYear = AcademyConversionProject.RevenueCarryForwardAtEndMarchCurrentYear,
+				ProjectedRevenueBalanceAtEndMarchNextYear = AcademyConversionProject.ProjectedRevenueBalanceAtEndMarchNextYear,
+				CapitalCarryForwardAtEndMarchCurrentYear = AcademyConversionProject.CapitalCarryForwardAtEndMarchCurrentYear,
+				CapitalCarryForwardAtEndMarchNextYear = AcademyConversionProject.CapitalCarryForwardAtEndMarchNextYear,
+				SchoolBudgetInformationAdditionalInformation = AcademyConversionProject.SchoolBudgetInformationAdditionalInformation,
+				SchoolBudgetInformationSectionComplete = AcademyConversionProject.SchoolBudgetInformationSectionComplete,
+				SchoolPupilForecastsAdditionalInformation = AcademyConversionProject.SchoolPupilForecastsAdditionalInformation,
+				KeyStagePerformanceTablesAdditionalInformation = AcademyConversionProject.KeyStagePerformanceTablesAdditionalInformation,
+				ProjectNote = (!string.IsNullOrEmpty(AcademyConversionProject.ProjectNoteSubject) || !string.IsNullOrEmpty(AcademyConversionProject.ProjectNoteBody)) ?
+					new UpdateProjectNote
+					{
+						Subject = AcademyConversionProject.ProjectNoteSubject, Note = AcademyConversionProject.ProjectNoteBody, Author = ""
+					} : null
+			};

```

Comment:
Same here I would leave UpdateAcademyConversionProjectPageModel just for updating an ACP entity and have a different page model for project notes